Github Issue Reader/ViewModels/IssueViewModel.swift

```diff
@@ -16,10 +16,14 @@ class IssueViewModel {
          let url = "https://api.github.com/repos/\(organization)/\(repo)/issues"
          NetworkingManager.shared.request(url, type: [Issue].self) { [weak self] response in
              DispatchQueue.main.async { [weak self] in
+                 guard let self = self else { return }
                  switch response {
                  case .success(let response):
-                     self?.issues.append(contentsOf: response)
-                     completion(self?.issues)
+                     // O(n^2) but we only get 30 issues at a time so yolo
+                     for issue in response where !self.issues.contains(issue) {
```

Comment:
Primarily because `UICollectionViewDiffableDataSource` needs all the items to be unique for reordering support. SwiftUI tackles this with `Identifiable` but this uses `Hashable`. I added this because opening, dismissing and opening again would crash with duplicates. 

I've addressed this whole thing by create a new IssuesViewModel each time we lookup the issues